.github/pull_request_template.md

```diff
@@ -0,0 +1,23 @@
+<!-- This comments are hidden when you submit the PR,
+so you do not need to remove them!
+
+**Note: If this PR closes a JIRA ticket, make sure the title
+starts with the JIRA issue number, for example 
+JP-123: <Fix a bug>
+**
+-->
+
+Closes #
+
+Closes JP-
```

Comment:
This is a minor nit, but many times when we merge a PR it does not technically "close" a JP ticket. It only sets it as resolved and sets the status to "Ready to test". Once someone else has tested and verified it works, then the JP ticket gets closed. So we might want to consider using a different word here, like "Fixes" or "Resolves".